cpu/mips_pic32_common/doc.txt

```diff
@@ -2,6 +2,8 @@
  * @defgroup        cpu_mips_pic32_common Microchip MIPS common
  * @ingroup         cpu
  * @brief           Microchip MIPS common
+ * @deprecated Will not be available after the 2022.07 release. This includes
```

Comment:
Do you mind also adding the notice to the documentation of both boards?

Reply:
Done